catkin_ws/src/sim/launch/sim.launch

```diff
@@ -18,23 +18,22 @@
     <arg name="vision" default="false" />
 
     <param name="sim" value="true" />
-
+    <param name="q_imunominal_imu_w" value="1.0" />
+    <param name="q_imunominal_imu_x" value="0.0" />
+    <param name="q_imunominal_imu_y" value="0.0" />
+    <param name="q_imunominal_imu_z" value="0.0"/>
+
+    <param name="q_dvl_auv_w" value="0.0" />
```

Comment:
does this work? wouldn't the state estimation param values overwrite these ones if it's launched after

Reply:
no state estimation only does that if the sim arg is false

Reply:
oh ok i see thats gucci then